src/ensembl/src/content/app/browser/browser-image/BrowserImage.tsx

```diff
@@ -68,7 +78,7 @@ export type BrowserImageProps = {
 };
 
 type BpaneOutPayload = {
-  bumper?: BrowserNavStates;
+  bumper?: boolean[];
```

Comment:
This type has now become too unspecific — we are losing both the information that there are six of these booleans here, and what these booleans represent.

A labelled tuple type would be a better fit:

```
[
  top: boolean,
  right: boolean,
  bottom: boolean,
  left: boolean,
  zoomOut: boolean,
  zoomIn: boolean
]
```

it's up to you whether you want to define this type here inline (ok if we are not reusing it) or define it as a separate type (might help if we are reusing it, e.g. when we are generating a payload to send back to genome browser)

Reply:
I have defined it as a separate type for now 👍